utils.groovy

```diff
@@ -109,7 +108,9 @@ def bump_builds_json(stream, buildid, arch, s3_stream_dir) {
             mkdir -p builds
             cp \$TMPD/builds/builds.json builds/builds.json
         fi
-        aws s3 cp --cache-control=max-age=300 --acl=public-read builds/builds.json s3://${s3_stream_dir}/builds/builds.json
+        export AWS_CONFIG_FILE=\${AWS_BUILD_UPLOAD_CONFIG}
+        aws s3 cp --cache-control=max-age=300    \
+            --acl=public-read builds/builds.json s3://${s3_stream_dir}/builds/builds.json
```

Comment:
I'd prefer we split this into its own `shwrapWithAWSBuildUploadCredentials` instead and leave the main on as `shwrap`.

Reply:
ok fixed.